utilities/PerformanceTesting/GetAverages.py

```diff
@@ -0,0 +1,79 @@
+import sys
```

Comment:
I guess what I meant by machine readable is that it should be in a well-defined format that already has a range of parsers that we can use. If it output json for example, it would be trivially easy for other tools to use that file without having to understand a proprietary text based format. `GetAverages.py` would also be much, much simpler because we could boil all the parsing code down to a single line of python.